### DISCUSSION TITLE
Account email change from android app

### DISCUSSION THREAD

**Discussion by H3n-beep** [simple-login/app#646] [Feature Requests]
Would be pretty good to have the ability to change the account main email so resetting password etc could be done on a different email

**Comment by nguyenkims** [comment]
@ntnhon we have indeed missed this setting while creating the settings on mobile app :).

**Reply by H3n-beep** [reply]
Well i really wish this option could make its way to the mobile app

**Comment by ntnhon** [comment]
Hello, I'll make that option for the new version that I'm working on.

**Reply by H3n-beep** [reply]
Hello and thanks for the inclusion of this setting
